__tests__/nc_news_db.test.js

```diff
@@ -254,6 +254,109 @@ describe("POST /api/articles/:article_id/comments", () => {
 	});
 });
 
+describe("PATCH /api/articles/:article_id", () => {
+	let originalVoteCount;
+
+	beforeAll(() => {
+		return db
+			.query("SELECT votes FROM articles WHERE article_id = 1")
+			.then((result) => {
+				originalVoteCount = result.rows[0].votes;
+			});
+	});
+
+	it("status: 200, should successfully increment the vote count", () => {
+		const input = { inc_votes: 1 };
+		return request(app)
+			.patch("/api/articles/1")
+			.send(input)
+			.expect(200)
+			.then(({ body }) => {
+				const article = body;
+				expect(article).toHaveProperty("votes", originalVoteCount + 1);
+			});
+	});
+
+	it("status: 200, should successfully decrease the vote count", () => {
+		const input = { inc_votes: -10 };
+		return request(app)
+			.patch("/api/articles/1")
+			.send(input)
+			.expect(200)
+			.then(({ body }) => {
+				const article = body;
+				expect(article).toHaveProperty("votes", originalVoteCount - 10);
+			});
+	});
+
+	it("status: 200, vote remains unchanged when increment counter is 0", () => {
+		const input = { inc_votes: 0 };
+		return request(app)
+			.patch("/api/articles/1")
+			.send(input)
+			.expect(200)
+			.then(({ body }) => {
+				const article = body;
+				expect(article).toHaveProperty("votes", originalVoteCount);
+			});
```

Comment:
For all your 200 tests I would change your originalVoteCount to the actual number so without the query above!